data/styles/clear/include/Icons.mapcss

```diff
@@ -448,6 +448,7 @@ node|z16-[historic=wreck],
 node|z16-[historic=locomotive],
 node|z16-[historic=boundary_stone],
 area|z16-[historic=boundary_stone],
+node|z16-[boundary=marker],
```

Comment:
Its a quite minor micro-mapped feature, so I think it should be visible starting at z18- or z19- even.

(`historic=boundary_stone` is visible earlier because its historic and hence is kinda touristic attraction)